km/km_coredump.h

```diff
@@ -172,18 +178,18 @@ typedef struct km_nt_event {
 } km_nt_event_t;
 
 // eventfd (epoll_create)
-typedef struct km_nt_eventfd {
+typedef struct km_nt_epollfd {
    Elf64_Word size;         // Size of record
    Elf64_Word fd;           // Open event fd
+   Elf64_Word dev;          // Device number - for dup detection
+   Elf64_Word ino;          // Inode number
    Elf64_Word flags;        // flags
    Elf64_Word event_size;   // size of event records that follow
    Elf64_Word nevent;       // number of event records that follow
-} km_nt_eventfd_t;
+   // Followed by event records km_nt_event_t
```

Comment:
add **km_nt_event_t nt_event[0];**

Reply:
makes code easier to write.

Reply:
I ws thinking about it but decided against it for this PR. There are many code improvements like doing `typedef enum` instead of sets of #define and such. Another PR for that